### ISSUE TITLE
Authentication failed

### ISSUE THREAD

**Issue by psycholyzern** [pritunl/pritunl#265]
Suddenly all user can't connect due to "Authentication faled" in Openvpn Connect. I deleted and recreate the users but still got authentication failed even I typed the password right.

I restarted pritunl and mongod, but no luck.

**Comment by zachhuff386** [comment]
Not enough information

**Comment by psycholyzern** [comment]
What information you needed, @zachhuff386? T_T

**Comment by psycholyzern** [comment]
Ok, this is the log.
https://www.dropbox.com/s/62vvds4retrq18a/pritunl_log.tar?dl=0

**Comment by psycholyzern** [comment]
@zachhuff386 should I open a new issue with the log or what? Im really lost here. Same.issue happened again after I reinstalled. 
Im now using DO 512MB.
At least say something, like you gonna ignore question like this or what? 

Thanks in advance.

**Comment by zachhuff386** [comment]
I need the logs in plain text in the issue comment.

**Comment by psycholyzern** [comment]
The log itself is 15361 lines. Would be ok if I posted here? 
That is a huge log.

**Comment by psycholyzern** [comment]
@zachhuff386 ugghh. I reinstalled on a new server and still get authentication failed error after a day of use?
Can I know if you are still interested to help me?
If yes which log you need?

**Comment by freuende** [comment]
i have the same issue.. at the beginning will be fine however after running couple of hours its started and error

**Comment by zachhuff386** [comment]
What is the output of the command `pritunl logs`

**Comment by psycholyzern** [comment]
That hell lot of lines.
Btw, there are too many of this:

```
restless-forest-3555][2016-02-29 23:19:21,308][ERROR] socket.error 'The write operation timed out'
Traceback (most recent call last):
  File "/usr/lib/pritunl/local/lib/python2.7/site-packages/pritunl/wsgiserver/wsgiserver2.py", line 1359, in communicate
    req.respond()
  File "/usr/lib/pritunl/local/lib/python2.7/site-packages/pritunl/wsgiserver/wsgiserver2.py", line 870, in respond
    self.server.gateway(self).respond()
  File "/usr/lib/pritunl/local/lib/python2.7/site-packages/pritunl/wsgiserver/wsgiserver2.py", line 2296, in respond
    self.write(chunk)
  File "/usr/lib/pritunl/local/lib/python2.7/site-packages/pritunl/wsgiserver/wsgiserver2.py", line 2360, in write
    self.req.write(chunk)
  File "/usr/lib/pritunl/local/lib/python2.7/site-packages/pritunl/wsgiserver/wsgiserver2.py", line 918, in write
    self.conn.wfile.sendall(chunk)
  File "/usr/lib/pritunl/local/lib/python2.7/site-packages/pritunl/wsgiserver/wsgiserver2.py", line 1012, in sendall
    bytes_sent = self.send(data)
  File "/usr/lib/pritunl/local/lib/python2.7/site-packages/pritunl/wsgiserver/wsgiserver2.py", line 1019, in send
    bytes_sent = self._sock.send(data)
  File "/usr/lib/python2.7/ssl.py", line 298, in send
    v = self._sslobj.write(data)
SSLError: The write operation timed out
```

I also opened logs menu in control panel, and it show about 15351 lines of error log.
Too much of them, I cant Identify if there is any other error other than I posted above.

**Comment by psycholyzern** [comment]
Btw, Im using 1GB RAM Digitalocean server.

**Comment by psycholyzern** [comment]
This means something?
The log is too long to be posted here. So I cut some.

```
[restless-forest-3555] Fri Mar  4 06:51:43 2016 TCP connection established with [AF_INET6]::ffff:128.199.194.173:57291
[restless-forest-3555] Fri Mar  4 06:51:43 2016 ::ffff:128.199.194.173 Connection reset, restarting [0]
[restless-forest-3555] Fri Mar  4 06:51:45 2016 TCP connection established with [AF_INET6]::ffff:128.199.194.173:57292
[restless-forest-3555] Fri Mar  4 06:51:45 2016 ::ffff:128.199.194.173 Connection reset, restarting [0]
[restless-forest-3555] Fri Mar  4 06:52:08 2016 TCP connection established with [AF_INET6]::ffff:5.45.64.228:46847
[restless-forest-3555] Fri Mar  4 06:52:08 2016 ::ffff:5.45.64.228 Connection reset, restarting [-1]
[restless-forest-3555] Fri Mar  4 06:59:27 2016 TCP connection established with [AF_INET6]::ffff:128.199.194.173:57361
[restless-forest-3555] Fri Mar  4 06:59:27 2016 ::ffff:128.199.194.173 Connection reset, restarting [0]
[restless-forest-3555] Fri Mar  4 06:59:29 2016 TCP connection established with [AF_INET6]::ffff:128.199.194.173:57364
[restless-forest-3555] Fri Mar  4 06:59:29 2016 ::ffff:128.199.194.173 TLS Error: Unroutable control packet received from [AF_INET6]::ffff:128.199.194.173:57364 (si=3 op=P_ACK_V1)
[restless-forest-3555] Fri Mar  4 06:59:29 2016 ::ffff:128.199.194.173 Fatal TLS error (check_tls_errors_co), restarting
[restless-forest-3555] Fri Mar  4 06:59:34 2016 TCP connection established with [AF_INET6]::ffff:128.199.194.173:57366
[restless-forest-3555] Fri Mar  4 06:59:34 2016 ::ffff:128.199.194.173 Connection reset, restarting [0]
[restless-forest-3555] Fri Mar  4 06:59:39 2016 TCP connection established with [AF_INET6]::ffff:128.199.194.173:57368
[restless-forest-3555] Fri Mar  4 06:59:39 2016 ::ffff:128.199.194.173 Connection reset, restarting [0]
[restless-forest-3555] Fri Mar  4 07:17:28 2016 TCP connection established with [AF_INET6]::ffff:5.45.64.228:4914
[restless-forest-3555] Fri Mar  4 07:17:28 2016 ::ffff:5.45.64.228 write TCPv6_SERVER: Connection reset by peer (code=104)
[restless-forest-3555] Fri Mar  4 07:17:28 2016 ::ffff:5.45.64.228 Connection reset, restarting [0]
[restless-forest-3555] Fri Mar  4 07:23:14 2016 COM> SUCCESS: client-auth command succeeded
[restless-forest-3555] Fri Mar  4 07:39:48 2016 TCP connection established with [AF_INET6]::ffff:128.199.194.173:57942
[restless-forest-3555] Fri Mar  4 07:39:48 2016 ::ffff:128.199.194.173 TLS Error: Unroutable control packet received from [AF_INET6]::ffff:128.199.194.173:57942 (si=3 op=P_ACK_V1)
[restless-forest-3555] Fri Mar  4 07:39:48 2016 ::ffff:128.199.194.173 Fatal TLS error (check_tls_errors_co), restarting
[restless-forest-3555] Fri Mar  4 07:39:53 2016 TCP connection established with [AF_INET6]::ffff:128.199.194.173:57943
[restless-forest-3555] Fri Mar  4 07:39:54 2016 ::ffff:128.199.194.173 TLS Error: Unroutable control packet received from [AF_INET6]::ffff:128.199.194.173:57943 (si=3 op=P_ACK_V1)
[restless-forest-3555] Fri Mar  4 07:39:54 2016 ::ffff:128.199.194.173 Fatal TLS error (check_tls_errors_co), restarting
[restless-forest-3555] Fri Mar  4 07:39:59 2016 TCP connection established with [AF_INET6]::ffff:128.199.194.173:57944
[restless-forest-3555] Fri Mar  4 07:39:59 2016 ::ffff:128.199.194.173 TLS Error: Unroutable control packet received from [AF_INET6]::ffff:128.199.194.173:57944 (si=3 op=P_ACK_V1)
[restless-forest-3555] Fri Mar  4 07:39:59 2016 ::ffff:128.199.194.173 Fatal TLS error (check_tls_errors_co), restarting
[restless-forest-3555] Fri Mar  4 07:40:04 2016 TCP connection established with [AF_INET6]::ffff:128.199.194.173:57945
[restless-forest-3555] Fri Mar  4 07:40:05 2016 ::ffff:128.199.194.173 TLS Error: Unroutable control packet received from [AF_INET6]::ffff:128.199.194.173:57945 (si=3 op=P_ACK_V1)
[restless-forest-3555] Fri Mar  4 07:40:05 2016 ::ffff:128.199.194.173 Fatal TLS error (check_tls_errors_co), restarting
[restless-forest-3555] Fri Mar  4 07:40:12 2016 TCP connection established with [AF_INET6]::ffff:128.199.194.173:57946
[restless-forest-3555] Fri Mar  4 07:40:13 2016 ::ffff:128.199.194.173 TLS Error: Unroutable control packet received from [AF_INET6]::ffff:128.199.194.173:57946 (si=3 op=P_ACK_V1)
[restless-forest-3555] Fri Mar  4 07:40:13 2016 ::ffff:128.199.194.173 Fatal TLS error (check_tls_errors_co), restarting
[restless-forest-3555] Fri Mar  4 07:40:18 2016 TCP connection established with [AF_INET6]::ffff:128.199.194.173:57947
[restless-forest-3555] Fri Mar  4 07:40:18 2016 ::ffff:128.199.194.173 TLS Error: Unroutable control packet received from [AF_INET6]::ffff:128.199.194.173:57947 (si=3 op=P_ACK_V1)
[restless-forest-3555] Fri Mar  4 07:40:18 2016 ::ffff:128.199.194.173 Fatal TLS error (check_tls_errors_co), restarting
[restless-forest-3555] Fri Mar  4 07:40:24 2016 TCP connection established with [AF_INET6]::ffff:128.199.194.173:57950
[restless-forest-3555] Fri Mar  4 07:40:24 2016 ::ffff:128.199.194.173 TLS Error: Unroutable control packet received from [AF_INET6]::ffff:128.199.194.173:57950 (si=3 op=P_ACK_V1)
[restless-forest-3555] Fri Mar  4 07:40:24 2016 ::ffff:128.199.194.173 Fatal TLS error (check_tls_errors_co), restarting
[restless-forest-3555] Fri Mar  4 07:40:31 2016 TCP connection established with [AF_INET6]::ffff:128.199.194.173:57951
[restless-forest-3555] Fri Mar  4 07:40:32 2016 ::ffff:128.199.194.173 TLS Error: Unroutable control packet received from [AF_INET6]::ffff:128.199.194.173:57951 (si=3 op=P_ACK_V1)
[restless-forest-3555] Fri Mar  4 07:40:32 2016 ::ffff:128.199.194.173 Fatal TLS error (check_tls_errors_co), restarting
[restless-forest-3555] Fri Mar  4 07:40:37 2016 TCP connection established with [AF_INET6]::ffff:128.199.194.173:57952
[restless-forest-3555] Fri Mar  4 07:40:37 2016 ::ffff:128.199.194.173 TLS Error: Unroutable control packet received from [AF_INET6]::ffff:128.199.194.173:57952 (si=3 op=P_ACK_V1)
[restless-forest-3555] Fri Mar  4 07:40:37 2016 ::ffff:128.199.194.173 Fatal TLS error (check_tls_errors_co), restarting
[restless-forest-3555] Fri Mar  4 07:40:42 2016 TCP connection established with [AF_INET6]::ffff:128.199.194.173:57953
[restless-forest-3555] Fri Mar  4 07:40:42 2016 ::ffff:128.199.194.173 TLS Error: Unroutable control packet received from [AF_INET6]::ffff:128.199.194.173:57953 (si=3 op=P_ACK_V1)
[restless-forest-3555] Fri Mar  4 07:40:42 2016 ::ffff:128.199.194.173 Fatal TLS error (check_tls_errors_co), restarting
[restless-forest-3555] Fri Mar  4 07:41:29 2016 TCP connection established with [AF_INET6]::ffff:128.199.194.173:57960
[restless-forest-3555] Fri Mar  4 07:41:34 2016 COM> SUCCESS: client-deny command succeeded
[restless-forest-3555] Fri Mar  4 07:41:34 2016 ERROR User auth failed "Unable to assign ip address"
[restless-forest-3555] Fri Mar  4 07:41:34 2016 ::ffff:128.199.194.173 [56d39c99f747861eedd05424] Peer Connection Initiated with [AF_INET6]::ffff:128.199.194.173:57960
[restless-forest-3555] Fri Mar  4 07:41:34 2016 ::ffff:128.199.194.173 Connection reset, restarting [0]
[restless-forest-3555] Fri Mar  4 07:41:45 2016 TCP connection established with [AF_INET6]::ffff:128.199.194.173:57969
[restless-forest-3555] Fri Mar  4 07:41:46 2016 ::ffff:128.199.194.173 TLS Error: Unroutable control packet received from [AF_INET6]::ffff:128.199.194.173:57969 (si=3 op=P_ACK_V1)
[restless-forest-3555] Fri Mar  4 07:41:46 2016 ::ffff:128.199.194.173 Fatal TLS error (check_tls_errors_co), restarting
[restless-forest-3555] Fri Mar  4 07:41:51 2016 TCP connection established with [AF_INET6]::ffff:128.199.194.173:57970
[restless-forest-3555] Fri Mar  4 07:42:03 2016 COM> SUCCESS: client-deny command succeeded
[restless-forest-3555] Fri Mar  4 07:42:03 2016 ERROR User auth failed "Unable to assign ip address"
[restless-forest-3555] Fri Mar  4 07:42:03 2016 ::ffff:128.199.194.173 [56d39c99f747861eedd05424] Peer Connection Initiated with [AF_INET6]::ffff:128.199.194.173:57970
[restless-forest-3555] Fri Mar  4 07:42:04 2016 ::ffff:128.199.194.173 Connection reset, restarting [0]
[restless-forest-3555] Fri Mar  4 07:42:12 2016 TCP connection established with [AF_INET6]::ffff:128.199.194.173:57972
[restless-forest-3555] Fri Mar  4 07:42:13 2016 ::ffff:128.199.194.173 TLS Error: Unroutable control packet received from [AF_INET6]::ffff:128.199.194.173:57972 (si=3 op=P_ACK_V1)
[restless-forest-3555] Fri Mar  4 07:42:13 2016 ::ffff:128.199.194.173 Fatal TLS error (check_tls_errors_co), restarting
[restless-forest-3555] Fri Mar  4 07:42:20 2016 TCP connection established with [AF_INET6]::ffff:128.199.194.173:57973
[restless-forest-3555] Fri Mar  4 07:42:29 2016 COM> SUCCESS: client-deny command succeeded
[restless-forest-3555] Fri Mar  4 07:42:29 2016 ERROR User auth failed "Unable to assign ip address"
[restless-forest-3555] Fri Mar  4 07:42:29 2016 ::ffff:128.199.194.173 [56d39c99f747861eedd05424] Peer Connection Initiated with [AF_INET6]::ffff:128.199.194.173:57973
[restless-forest-3555] Fri Mar  4 07:42:29 2016 ::ffff:128.199.194.173 Connection reset, restarting [0]
[restless-forest-3555] Fri Mar  4 07:42:49 2016 TCP connection established with [AF_INET6]::ffff:5.45.64.228:51196
[restless-forest-3555] Fri Mar  4 07:42:49 2016 ::ffff:5.45.64.228 Connection reset, restarting [-1]
```

**Comment by zachhuff386** [comment]
The SSL write timeout error above has been fixed in v1.18.906.35
